coralproject/talk
# Cleanup on email verification

## What does this PR do?

- Changes "confirm" copy to "verify" and updates the endpoints so there's no confusion.
- updates handling of the redirect uri to use the iframe parent url
- adds copy to the Sign Up dialog to let a user know if they need to check their email to verify

## How do I test this PR?

- Sign Up a new user with email verification turned on
- you should now see some copy about how you have to verify your email (this should not be visible if you don't have email verification enabled)
- there shouldn't be any "confirm" copy anywhere. It should have been changed to "verify"
